playbooks/robusta_playbooks/prometheus_simulation.py

```diff
@@ -100,6 +105,8 @@ def prometheus_alert(event: ExecutionBaseEvent, prometheus_event_data: Prometheu
                     "endsAt": datetime.now(),
                     "startsAt": datetime.now(),
                     "generatorURL": prometheus_event_data.generator_url,
+                    "runbookURL": prometheus_event_data.runbook_url,
```

Comment:
@arikalon1 the meaning is that I wanted to test the `runbookURL` with Prometheus simulation that Robusta provides. That value is put into the `annotations` dict into the `prometheus_simulation.py` code.

A real prometheus alert can have the `runbookURL` into its annotations, and so imho this field allows to test the feature.

If it's ok for you, we could leave this here, what do you think?

Reply:
Can you explain how you use it?

We take the alert input, and put it in this class: (the schema is from [here](https://prometheus.io/docs/alerting/latest/notifications/#alert))
```
class PrometheusAlert(BaseModel):
    endsAt: datetime
    generatorURL: str
    startsAt: datetime
    fingerprint: Optional[str] = ""
    status: str
    labels: Dict[Any, Any]
    annotations: Dict[Any, Any]
```

So, I think the `runbook_url' in that location, will not be included in the notification at all.
Am I wrong on that?

Reply:
@arikalon1 Yes, I've overlooked that code line, even my explanation above is wrong. We're using `runbook_url` within the annotation, so that line is redundant as you correctly pointed out.

I've removed it, thanks